cmd/system-probe/config/config.go

```diff
@@ -170,8 +171,9 @@ func load(configPath string) (*Config, error) {
 		LogLevel:  cfg.GetString(key(spNS, "log_level")),
 		DebugPort: cfg.GetInt(key(spNS, "debug_port")),
 
-		StatsdHost: aconfig.GetBindHost(),
-		StatsdPort: cfg.GetInt("dogstatsd_port"),
+		StatsdHost:            aconfig.GetBindHost(),
+		StatsdPort:            cfg.GetInt("dogstatsd_port"),
+		SystemProbeExpVarPort: cfg.GetInt(key(spNS, "expvar_port")),
```

Comment:
tiny nit: since this is the system-probe config, `SystemProbe` is an unnecessary prefix here & IMO should be removed